public/app/features/alerting/unified/components/rule-viewer/Actions.tsx

```diff
@@ -101,15 +121,3 @@ const ExportMenuItem = ({ identifier }: PropsWithIdentifier) => {
 
   return <Menu.Item key="with-modifications" label="With modifications" icon="file-edit-alt" url={url} />;
 };
-
-const EditButton = ({ identifier }: PropsWithIdentifier) => {
```

Comment:
This used to be a combination of the edit button + the dropdown, but it made more sense to split it so this is only the dropdown items